doc/usage/quickstart.rst

```diff
@@ -26,9 +43,6 @@ configuration values from a few questions it asks you. To use this, run:
 
    $ sphinx-quickstart
 
-There is also an automatic "API documentation" generator called
-:program:`sphinx-apidoc`; see :doc:`/man/sphinx-apidoc` for details.
```

Comment:
I removed this reference since I didn't think it was relevant for a quickstart guide. Many people won't care about this tool.

Reply:
+1; I think it is advanced usage.